tgui/packages/tgui/interfaces/CharacterPreferences/SavesTab.tsx

```diff
@@ -0,0 +1,51 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../../backend';
+import { Box, Button, Divider, LabeledList, Section } from '../../components';
+import { CharacterPreferencesData } from './type';
+
+export const SavesTab = (_props, context) => {
+  const { act, data } = useBackend<CharacterPreferencesData>(context);
+
+  return (
+    <Section title="Cloud Saves">
+      {data.cloudSaves ? (
+        <Fragment>
```

Comment:
It is fine to use empty tags `<>`, they also do not require importing Fragments.